.circleci/config.yml

```diff
@@ -99,19 +99,30 @@ jobs:
     docker:
       - image: cimg/node:16.15-browsers
     steps:
-      - checkout
-      - browser-tools/install-chrome
-      - browser-tools/install-chromedriver
-
-      #install-packages documentation: https://circleci.com/developer/orbs/orb/circleci/node#commands-install-packages
-      - node/install-packages:
-          override-ci-command: npm install
-          app-dir: ~/project/front-end/
-
+      - setup_remote_docker:
+          docker_layer_caching: true
       - run:
-          name: run e2e tests
-          command: node --max_old_space_size=4000 ./node_modules/@angular/cli/bin/ng e2e
-          working_directory: ~/project/front-end/
+          name: init fecfile-web-api
+          command: |
+            cd ~
+            rm -rf .git fecfile-web-api
+            git clone https://github.com/fecgov/fecfile-web-api.git
+            cd fecfile-web-api
+            git checkout feature/${E2E_BRANCH_NAME}
+            docker-compose down
+            DB_DOCKERFILE="Dockerfile-e2e" WORKER_DOCKERFILE="Worker_Dockerfile-e2e" API_DOCKERFILE="Dockerfile-e2e" FECFILE_TEST_DB_NAME="postgres"  DJANGO_SECRET_KEY=${E2E_DJANGO_SECRET_KEY} DATABASE_URL=${E2E_DATABASE_URL} docker-compose up -d
+            docker container run --network container:fecfile-api \
+              docker.io/jwilder/dockerize \
+              -wait http://localhost:8080/api/v1/ \
+              -wait-retry-interval 2s \
+              -timeout 60s; \
+            docker container run -e E2E_BRANCH_NAME=${E2E_BRANCH_NAME} -e CYPRESS_EMAIL=${CYPRESS_EMAIL} -e CYPRESS_COMMITTEE_ID=${CYPRESS_COMMITTEE_ID} -e CYPRESS_PASSWORD=${CYPRESS_PASSWORD} --network container:fecfile-api cypress/browsers:node18.12.0-chrome107 /bin/bash -c '\
+              cd ~; \
+              git clone https://github.com/fecgov/fecfile-web-app.git; \
```

Comment:
I think line 121-123 can be replaced with a `-checkout`
less inline bash scripting, and it will be it's own circleci step in the job

Reply:
@toddlees the trouble is that circleci commands are not available inside the container context (only bash commands)  and I can't really seem to find any workaround for that (if you know of any I am all ears though).  I was still however able to refactor into multiple run steps and will should give us more a more granular view of the run.